### PR TITLE
Fixes broken distributions view

### PR DESCRIPTION
**Pull request by ChrisBr**

### CODE REVIEW

**Comment by hennevogel** [comment]
Are you sure dot is the only problematic character? :-)

**Comment by ChrisBr** [comment]
@hennevogel updated ...

File: src/api/app/helpers/webui/webui_helper.rb [hunk 1]
```diff
@@ -388,4 +388,8 @@ def escape_nested_list(list)
       "']".html_safe
     }.join(",\n").html_safe
   end
+
+  def replace_meta_characters(input)
+  input.sub("!\"#$%&'()*+,./:;<=>?@[\\]^`{|}~", '_')
```
Comment (mdeniz) [inline src/api/app/helpers/webui/webui_helper.rb]: looks like this is wrongly indented

Comment (hennevogel) [inline src/api/app/helpers/webui/webui_helper.rb]: Have you guessed this or is this based on something? :-)

File: src/api/app/views/webui/repositories/distributions.html.erb [hunk 2]
```diff
@@ -8,27 +8,31 @@
...
-                  <% end %>
-                  <%= form_tag({ action: :destroy }, remote: true, class: 'hidden', id: 'repo_' + distribution['reponame'] + '_destroy') do %>
-                    <%= hidden_field_tag 'project', @project.name %>
-                    <%= hidden_field_tag 'target', distribution['reponame'] %>
-                  <% end %>
               </span>
           <% end -%>
-        </p>
+         </p>
+      <% list.each do |distribution| %>
```
Comment (bgeuken) [inline src/api/app/views/webui/repositories/distributions.html.erb]: I must say that I don't really like that we loop through distributions twice. Unfortunately I don't know a quick way to solve this. So we probably have to live with it.

File: src/api/app/helpers/webui/webui_helper.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/app/helpers/webui/webui_helper.rb]: @ChrisBr Please fix this so we can merge it:-)

Comment (ChrisBr) [inline src/api/app/helpers/webui/webui_helper.rb]: http://api.jquery.com/category/selectors/

**Comment by ChrisBr** [comment]
@mdeniz fixed indentation

**Comment by mdeniz** [comment]
LGTM :+1:

**Pull request merged**